### ISSUE TITLE
Reserved state in settlements is not working for payeefsp

### ISSUE THREAD

**Issue by sri-miriyala** [mojaloop/project#597]
**Summary**:
During the settlements process, When I tried updating the participants, PS_TRANSFERS_RECORDED working fine for both payerfsp and payeefsp. But PS_TRANSFERS_RESERVED is working only for the first partcipant but not for the other. Hence could not test the other states.

**Severity**:
HIGH
**Priority**:
Critical

**Expected Behavior**
1. When I update settlement by settlement participant, by providing settlement ID, participant ID, position account Id and state "PS_TRANSFERS_RESERVED" for payerfsp and payeefsp seperately
2. Payerfsp account should be changed to PS_TRANSFERS_RESERVED
3. Payeefsp account should be changed to PS_TRANSFERS_RESERVED
4. So that the entire settlement state should be PS_TRANSFERS_RESERVED as there are only payerfsp and payeefsp participants.

**Steps to Reproduce**
  1. When I update settlement by settlement participant, by providing settlement ID, participant ID, position account Id and state "PS_TRANSFERS_RESERVED" for payerfsp and payeefsp seperately
2. Payerfsp account changed to PS_TRANSFERS_RESERVED
3. Payeefsp account did not change but giving the below error:
```
{
"statusCode": 400,
"error": "Bad Request",
"message": {
"errorInformation":

{ "errorDescription": "Cannot read property 'TOPIC_TEMPLATES' of undefined" }
```
**Below is the request to be used in step1 of Steps to Reproduce:**
```
curl -X PUT http://test-central-settlement.mojaloop.live/v1/settlements/3 \
-H 'Content-Type: application/json' \
-d '{
"participants": [
 {
  "id": 4,
  "accounts": [
    { "id": 5, "reason": "test", "state": "PS_TRANSFERS_RESERVED" } ] }
]
}'
```

**Specifications**
  - Component (if known): Central-Settlement
  - Version: 4.4.1
  - Platform: AWS 'Dev'
  - Subsystem: Settlement Transfers
  - Type of testing: Functional
  - Bug found/raised by: Sri

**Notes**:
Further testing on settlements is blocked.

- Severity when opened: High
- Priority when opened: Critical

**Comment by sri-miriyala** [comment]
@ggrg updated step 3 to payeefsp. Sorry.. it was my mistake. But there is curl request in there, which I used to do different states of the settlement.

**Comment by ggrg** [comment]
@sri-miriyala, I was able to reproduce the issue and the problem is caused by missing configuration in default.json. @elnyry, please update config/default.json as requested in Slack. Thanks.

**Comment by elnyry** [comment]
Thanks Georgi. Raised this PR - https://github.com/mojaloop/helm/pull/137

**Comment by sri-miriyala** [comment]
@ggrg Thanks for quickly looking into it.

**Comment by elnyry** [comment]
PR is approved and merged now. Will release it soon (after which we can redeploy)

**Comment by elnyry** [comment]
Redeployed, ready to be validated

**Comment by ggrg** [comment]
Validated with request for NET_SETTLEMENT_RECIPIENT:
```
{
  "participants": [
    {
      "id": 4,
      "accounts": [
        {
          "id": 5,
          "state": "PS_TRANSFERS_RESERVED",
          "reason": "test 11",
          "externalReference": "aaa"
        }
      ]
    }
  ]
}
```
Outputs response:
```
{
    "id": 1,
    "state": "PS_TRANSFERS_RESERVED",
    "createdDate": "2019-02-06 21:27:32.167",
    "settlementWindows": [
        {
            "settlementWindowId": 1,
            "settlementWindowStateId": "PENDING_SETTLEMENT",
            "reason": "Georgi 35",
            "createdDate": "2019-02-06T16:43:53.000Z"
        }
    ],
    "participants": [
        {
            "id": 4,
            "accounts": [
                {
                    "id": 5,
                    "state": "PS_TRANSFERS_RESERVED",
                    "reason": "test 11",
                    "externalReference": "aaa",
                    "createdDate": "2019-02-06 21:27:32.167",
                    "netSettlementAmount": {
                        "amount": -6,
                        "currency": "USD"
                    }
                }
            ]
        }
    ]
}
```

**Comment by sri-miriyala** [comment]
@ggrg  thanks much.
Yes.. for me too it is working from the fix v5.1.0
Below is the screenshot that shows the participant accounts "SETTLED" 
Also I confirm that Settlement ID state and Settlement window states are also getting updated appropriately.
<img width="1040" alt="screen shot 2019-02-07 at 12 57 51 am" src="https://user-images.githubusercontent.com/38143240/52393389-ce2a6000-2a73-11e9-9af3-5b3c5b03dbad.png">

Hence closing this issue.
